modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -222,16 +222,25 @@ struct IEUnit {
     IE::ExecutableNetwork this_network;
     cv::gimpl::ie::wrap::Plugin this_plugin;
 
+    InferenceEngine::RemoteContext::Ptr rctx = nullptr;
+
     explicit IEUnit(const cv::gapi::ie::detail::ParamDesc &pp)
         : params(pp) {
+        InferenceEngine::ParamMap* ctx_params =
+                            cv::util::any_cast<InferenceEngine::ParamMap>(&params.context_config);
```

Comment:
so before making this cast at all, can you please check if `any` is not null?

Reply:
Done

Reply:
Under null I mean empty, but I see our `any` don't have `has_value` as defined in (std)[https://en.cppreference.com/w/cpp/utility/any/has_value]

Please add a follow-up task

Reply:
Added a task